Source/WTF/Scripts/Preferences/WebPreferencesInternal.yaml

```diff
@@ -572,6 +572,18 @@ LiveRangeSelectionEnabled:
     WebCore:
       default: false
 
+LiveTextForFindInPageEnabled:
```

Comment:
Nit - w.r.t. setting names, I've been largely avoiding names like "Live Text" that are directly tied to features on Apple platforms (though currently, it's true that it's only relevant on Apple platforms). What do you think about something like `TextRecognitionDuringFindInPageEnabled`?

Reply:
Fixed!